plugins/PostProcessingPlugin/scripts/PauseAtHeightforRepetier.py

```diff
@@ -67,6 +67,21 @@ def getSettingDataString(self):
                     "unit": "layers",
                     "type": "int",
                     "default_value": 0
+                },
+                ,
+                "custom_gcode_before_pause":
+                {
+                    "label": "GCODE Before Pause",
+                    "description": "Any custom GCODE to run before the pause, for example, M300 S300 P1000 to beep.",
+                    "type": "str",
+                    "default_value": "M300 S440 P200; M300 S660 P250; M300 S880 P300;"
```

Comment:
Please clear this to have no beeps. The example in the description is good, but this default will change the behaviour for what will be a bit unexpected for most people.
I think the most useful use case for this is to cool down the build plate or something to save power, or to perform a wipe on a wiping brush before continuing the print.